Code/Editor/AboutDialog.cpp

```diff
@@ -46,8 +47,16 @@ CAboutDialog::CAboutDialog(QString versionText, QString richTextCopyrightNotice,
                     CAboutDialog > QLabel#link { text-decoration: underline; color: #94D2FF; }");
 
     // Prepare background image
-    QImage backgroundImage(QStringLiteral(":/StartupLogoDialog/splashscreen_background_developer_preview.jpg"));
-    m_backgroundImage = QPixmap::fromImage(backgroundImage.scaled(m_enforcedWidth, m_enforcedHeight, Qt::IgnoreAspectRatio, Qt::SmoothTransformation));
+    QPixmap backgroundImage = QPixmap(QStringLiteral(":/StartupLogoDialog/splashscreen_background_developer_preview.jpg"));
+
+    qreal screenDpiFactor = QHighDpiScaling::factor(screen());
```

Comment:
Can we move this to an `AzQtComponents` or `AzToolsFramework` helper function somewhere, to avoid proliferating private Qt header includes downstream? I'm totally OK with the logic, I just want to harden us against the "can change at any time" nature of internal Qt API.

Reply:
Looking at it more, we might be able to move the image sizing itself into its own little helper and DRY this a little.

Reply:
Yup agreed, I will add a helper

Reply:
Added a helper.